src/Stepper/Step.js

```diff
@@ -81,7 +81,7 @@ export type Props = {
   /**
    * Define this step as optional.
```

Comment:
This should provide provide some indication that this property serves two purposes, marking the step as optional and defining the text that will be presented.  Perhaps a note similar to the one for the `value` prop in the [Tabs](https://material-ui-next.com/api/tabs/) component:

>The value of the currently selected Tab. If you don't want any selected Tab, you can set this property to false.